### ISSUE TITLE
Python Mediapipe built from source for GPU doesn't run on GPU

### ISSUE THREAD

**Issue by petremihaivalentin** [google/mediapipe#2079]
So I tried to build the MediaPipe Python Package from source for the GPU. I did the following steps:

From [this](https://google.github.io/mediapipe/getting_started/install.html):
  1. Installed Bazelisk
  2. Cloned MediaPipe repo
  3. Installed OpenCV and FFmpeg using option 1
  4. Did the 4th step for GPU support
  5. Ran the example with the command for GPU support -> I've verified that this ran on the GPU

From [this](https://google.github.io/mediapipe/getting_started/python.html), we did all the steps under Building MediaPipe Python Package

Then I tried running a simple script:

```python
import cv2
import mediapipe as mp
import numpy as np
import time
mp_drawing = mp.solutions.drawing_utils
mp_pose = mp.solutions.pose

def get_pose():
    start = time.time()
    pose = mp_pose.Pose(min_detection_confidence=0.5, min_tracking_confidence=0.5)
    end = time.time()
    print("MODEL LOADED: ", end - start)

    i = 100
    while i > 0:
        if i % 2 == 0:
            frame = cv2.imread("image.png")
        else:
            frame = cv2.imread("image2.png")
    
        image = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
        image.flags.writeable = False
        start = time.time()

        # Make detection
        results = pose.process(image)
        end = time.time()
        print("ELAPSED: ", end - start)


        # Recolor back to BGR
        image.flags.writeable = True
        image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)

        # Extract landmarks
        try:
            landmarks = results.pose_landmarks.landmark
            # print(landmarks)
        except:
                    pass
        
        
                # Render detections
                mp_drawing.draw_landmarks(image, results.pose_landmarks, mp_pose.POSE_CONNECTIONS,
                                        mp_drawing.DrawingSpec(color=(245,117,66), thickness=2, circle_radius=2),
                                        mp_drawing.DrawingSpec(color=(245,66,230), thickness=2, circle_radius=2)
                                        )
                i -= 1

if __name__ == '__main__':
    get_pose()

```

And I've noticed that this doesn't run on GPU.

I'm running everything on an Ubuntu 18.04/ Python 3.8.10.

Is there anything I've done wrong when installing? Thank you.

**Comment by sgowroji** [comment]
Hi @petremihaivalentin, Thank you for sharing the above details. What is the error are you facing while running on GPU?

**Comment by petremihaivalentin** [comment]
Hello @sgowroji, 
Thanks for your fast response. I got no error at runtime, but I tailed nvidia-smi and there was no activity on the GPU.

**Comment by petremihaivalentin** [comment]
Hello again,

I tried rebuilding from source the GPU python version by following [this](https://github.com/google/mediapipe/issues/2041#issuecomment-844129267).
Afterwards, I ran ```python3.8 setup.py install```. Now I get this error: 
![image](https://user-images.githubusercontent.com/49815020/120845060-f27da500-c578-11eb-8ece-931a54213032.png)

I have to mention that I had to comment out some ```remove_generated``` from the setup.py file for the build to finish successfully (the script deleted the __init__.py files and then errored out because it couldn't find them). Afterwards, I deleted manually the __init__.py files that the script was trying to delete. Now I'm stuck here.

What should I try?

Thanks
